packages/editor/src/components/assets/FileBrowser/FileBrowserContentPanel.tsx

```diff
@@ -130,7 +131,7 @@ const FileBrowserContentPanel: React.FC<FileBrowserContentPanelProps> = (props)
   const selectedDirectory = useHookstate(originalPath)
   const nestingDirectory = useHookstate(props.nestingDirectory || 'projects')
   const fileProperties = useHookstate<FileType | null>(null)
-
+  const { projectName } = useMutableState(EditorState)
```

Comment:
editor state is not the correct data to use here. we need to somehow get it from the file browser's state

Reply:
Changed to a component prop.